network-api/networkapi/settings.py

```diff
@@ -114,6 +119,12 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = root()
 
+# Basket client configuration
+BASKET_URL = env('BASKET_URL')
```

Comment:
The basket URL is set to use the staging environment of basket in env.default.

We dont need the basket API key at the moment, but I left it here as Kalob mentioned it might be a good idea in case we need it in the future. 

The subscribe method in env.default is defaulting to BASKET